view/material/Table.kt

```diff
@@ -75,21 +75,21 @@ object Table {
     private val CELL_PADDING_VERTICAL = 4.dp
 
     @Composable
-    private fun bgColor(i: Int): Color =
-        if (i % 2 == 0) Theme.studio.backgroundLight else Theme.studio.backgroundMedium
+    private fun bgColor(i: Int): Color = if (i % 2 == 0) Theme.studio.backgroundLight else Theme.studio.backgroundMedium
 
     @Composable
     fun <T> Layout(
         items: List<T>,
         modifier: Modifier = Modifier,
+        showHeaders: Boolean = true,
         rowHeight: Dp = ROW_HEIGHT,
         columnBorderSize: Dp = COLUMN_BORDER_SIZE,
         horCellPadding: Dp = CELL_PADDING_HORIZONTAL,
         verCellPadding: Dp = CELL_PADDING_VERTICAL,
         columns: List<Column<T>>
     ) {
         Column(modifier.border(1.dp, Theme.studio.border)) {
-            Header(rowHeight, columnBorderSize, horCellPadding, verCellPadding, columns)
+            if (showHeaders) Header(rowHeight, columnBorderSize, horCellPadding, verCellPadding, columns)
```

Comment:
We introduced `showHeaders` to `Table`, giving the option to not show a header row at all.